Assets/Src/AuthController/REST/REST Request Proxies/GoogleIdTokenRequestDtoProxy.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Src.AuthController.REST.REST_Request_Proxies
+{
+    [Serializable]
+    public class GoogleIdTokenRequestDtoProxy
+    {
+        public string client_id;
```

Comment:
Use pascal case for public members. Also, consider installing SonarLint for code style hints.